.golangci.yml

```diff
@@ -111,5 +111,8 @@ issues:
     - path: controlplane/pkg/monitor/
       linters:
         - dupl
```

Comment:
As I understood this PR should fixes code duplication. Can you remove this linter right?